test/rng_testsuite/extra_methods_tests/distributions_methods.pass.cpp

```diff
@@ -0,0 +1,269 @@
+                    }
+                    else
+                    {
+                        res0 = d1(engine, params2);
+                        res1 = d1(engine, params1);
+                    }
+                    dpstd_acc[offset * 2] = res0;
+                    dpstd_acc[offset * 2 + 1] = res1;
+                });
+            });
+        }
+        catch (sycl::exception const& e)
+        {
+            std::cout << "\t\tSYCL exception during generation\n"
+                      << e.what() << std::endl
+                      << "OpenCL status: " << e.get_cl_code() << std::endl;
+            return 1;
+        }
+
+        queue.wait_and_throw();
+        Distr distr;
+        // 0 and 1 are default parameters for uniform_int, uniform_real and normal distributions
+        check_params(distr, static_cast<typename Distr::scalar_type>(0), static_cast<typename Distr::scalar_type>(1));
+    }
+
+    return sum;
+}
+
+#endif // _ONEDPL_BACKEND_SYCL
+
+int
+main()
+{
+
+#if _ONEDPL_BACKEND_SYCL
+
+    std::int32_t err = 0;
+    std::int32_t global_err = 0;
+
+    std::cout << "---------------------------------------------------" << std::endl;
+    std::cout << "uniform_int_distribution<std::int32_t>" << std::endl;
+    std::cout << "---------------------------------------------------" << std::endl;
+    err += test<oneapi::dpl::uniform_int_distribution<std::int32_t>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 1>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 2>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 3>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 4>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 8>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::int32_t, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }
+    global_err += err;
+    err = 0;
+
+    std::cout << "---------------------------------------------------" << std::endl;
+    std::cout << "uniform_int_distribution<std::uint32_t>" << std::endl;
+    std::cout << "---------------------------------------------------" << std::endl;
+    err += test<oneapi::dpl::uniform_int_distribution<std::uint32_t>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 1>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 2>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 3>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 4>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 8>>>();
+    err += test<oneapi::dpl::uniform_int_distribution<sycl::vec<std::uint32_t, 16>>>();
+    if (err)
+    {
+        std::cout << "Test FAILED" << std::endl;
+    }

```

Comment:
In `support/utils.h` we have `EXPECT_TRUE` that can help you to process such cases, e.g. 
`EXPECT_TRUE(!err, "Test FAILED");`

Reply:
changed